ludwig/api.py

```diff
@@ -1649,6 +1649,15 @@ def set_logging_level(logging_level: int) -> None:
         else:
             set_disable_progressbar(False)
 
+    @property
+    def config(self) -> Dict[str, Any]:
+        return self.config_obj.to_dict()
+
+    @config.setter
+    def config(self, user_config: Dict[str, Any]):
```

Comment:
Perhaps the setter should be private, or at least have a warning in the doc string explaining its use.

Setting the config of an already-constructed LudwigModel could cause unexpected behavior